packages/statusbar/src/defaults/runningSessions.tsx

```diff
@@ -40,6 +40,8 @@ function RunningSessionsComponent(
         <TextItem source={props.kernels} />
         <IconItem source={'jp-StatusItem-kernel'} offset={{ x: 0, y: 2 }} />
       </GroupItem>
+      <span className="screenReaderOnly" aria-live="assertive" aria-atomic="true" id="arialive-status">
```

Comment:
Does this span show up in the visible appearance of the page? Are there any layout consequences of the `clip` property?

Reply:
No, this is a 0px element, but we cannot use "display: hidden" because screen readers ignore such elements.